sdk/core/Azure.Core/README.md

```diff
@@ -8,24 +8,27 @@ A more complete list of client libraries using Azure.Core can be found [here](ht
 Azure.Core allows client libraries to expose common functionality in a consistent fashion, 
 so that once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
 
-The main shared concepts of Azure.Core (and so Azure SDK libraries using Azure.Core) include:
-
-- Configuring service clients, e.g. configuring retries, logging.
-- Accessing HTTP response details.
-- Calling long-running operations (LROs).
-- Paging and asynchronous streams (```AsyncPageable<T>```) 
-- Exceptions for reporting errors from service requests in a consistent fashion.
-- Abstractions for representing Azure SDK credentials.
+## Getting started
 
-Below, you will find sections explaining these shared concepts in more detail.
-
-## Installing
 Typically, you will not need to install Azure.Core; 
 it will be installed for you when you install one of the client libraries using it. 
 In case you want to install it explicitly (to implement your own client library, for example), 
 you can find the NuGet package [here](https://www.nuget.org/packages/Azure.Core).
 
-## Usage Scenarios and Samples
+# Key concepts
```

Comment:
Should this be an H2?

Reply:
```suggestion
## Key concepts
```

Reply:
It should be a h2 https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#key-concepts